### ISSUE TITLE
Add to fastify benchmarks

### ISSUE THREAD

**Issue by styfle** [mafintosh/turbo-http#5]
There are a bunch of http libs that claim to be fast.
And there is a benchmark to [see who is fastest](https://github.com/fastify/fastify#benchmarks).

You should add `turbo-http` to the benchmarks repo and see how it measures up 😄 
https://github.com/fastify/benchmarks

**Comment by mhart** [comment]
I don't think this is really relevant – this module is a level below a web framework – it's essentially a replacement for `require('http')`... but even simpler than that. It's possible the frameworks listed there could run on this instead of the built-in node.js http module, but I imagine that's not necessarily its goal.

**Comment by styfle** [comment]
Good point!

Maybe this quote from the readme could show the source code for the benchmark and test harness to produce the results. For example, is this one test run or an average of 50 test runs?

> On my laptop I can serve simple hello world payloads at around 100k requests/seconds compared to 10k requests/second using node core.

**Comment by lukeed** [comment]
Not relevant. This is a HTTP server, as opposed to a server framework. Aka, this replaces `require('http')` but does not replace `require('express')`.

**Comment by styfle** [comment]
Looks like someone went ahead and did it anyway: https://github.com/fastify/benchmarks/pull/47

**Comment by tuananh** [comment]
@lukeed so sth like this should qualify? https://github.com/emkay/theodore

**Comment by lukeed** [comment]
@tuananh The framework, yes. But only when it's running on `require('http')`. The suite is benchmarking _frameworks_ and _framework overheads_, not HTTP servers.

**Comment by lukeed** [comment]
Otherwise things like [`fasthttp`](https://github.com/valyala/fasthttp) to the Fastify benchmark (which would be meaningless).

**Comment by tuananh** [comment]
@lukeed I get your point. I just don't understand why using `http` core module is a must when comparing, as long as it's a web framework. never mind.

**Comment by lukeed** [comment]
Because that's the most common module. Most people are using it, and all tests _are_ running with it. Turbo, UWS, etc are all upgrades. AKA, it's like showing up to a high school baseball game on a double dose of steroids -- it isn't fair.

Best to benchmark on native / "raw" because it's already the baseline. Then let people upgrade to their preferred HTTP alternative if/when they want to.

**Comment by styfle** [comment]
I understand now that it doesn't make sense to compare with fastify.
But it would make sense to compare with other HTTP frameworks.

@mafintosh How about adding it to https://www.techempower.com/benchmarks/ so that it can be compared to the node.js core http module and many other web frameworks?

**Comment by aichholzer** [comment]
See [this](https://github.com/mafintosh/turbo-http/pull/22).
**Disclaimer:** Although I am 100% with @lukeed on this one (and we've had this very conversation elsewhere on Github), some people might still find it nice and useful to be able to see some numbers.
